README.md

```diff
@@ -1,89 +1,110 @@
 # USDS website
 
-## Setup
+The USDS website is built with:
 
-### Built with
-
-- [Jekyll](https://jekyllrb.com/)
+- [Jekyll](https://jekyllrb.com/) via the [github-pages gem](https://rubygems.org/gems/github-pages)
+  - See [current dependencies](https://pages.github.com/versions/) for GitHub Pages
 - [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/)
+- [Gulp](https://gulpjs.com/) / [USWDS Gulp](https://github.com/uswds/uswds-gulp)
 
-### Prerequisites
+Before getting started, install the following on your system:
 
-- [Ruby >=2.6.5](https://www.ruby-lang.org/en/documentation/installation/)
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
 - [NPM](https://github.com/npm/cli)
 - [Gulp CLI](https://gulpjs.com/docs/en/getting-started/quick-start)
 
+## Install and run
 
-### Install & run
-
+### Initial setup
 ```
 $ npm install
 $ bundle install
-$ npm start (runs bundle exec jekyll serve --livereload)
+$ npm start
 ```
 
-## Content updates
+### Running and building
+#### Run locally & watch for changes
+```
+$ npm start
+~ or ~
+$ npm run serve
+```
+Builds production Jekyll site, spins it up at `localhost:4000`, and watches for changes. Compiles main stylesheet with Gulp and watches for changes in `assets/stylesheets/**/*.scss`.
 
-### Uploading images
-- [/images](https://github.com/usds/website/tree/master/images): This folder contains *editorial photos* and other images for People cards and pages, Project cards and pages, event logos for the Events cards and page, and other areas that are likely to change semi-frequently.
-- [/assets/img](https://github.com/usds/website/tree/master/assets/img): Site assets and evergreen images such as page banners/headers and vector graphics should be placed in the assets/img folder here. The images in this folder typically will not be subject to change.
+#### Production build
```

Comment:
```suggestion
#### Local build
```

why: i'm worried this would be interpreted as `JEKYLL_ENV=production`, which has side effects of analytics tracking

Reply:
Oh nice callout. Pulling "production" out is a good idea here.

Had added that because _technically_ what gets build locally is what goes into production on GitHub pages. But we're also not using the production environment flag specifically.